dd-sdk-android/build.gradle.kts

```diff
@@ -161,6 +162,7 @@ unMock {
     keep("android.view.Choreographer")
     keep("android.view.DisplayEventReceiver")
     keepStartingWith("org.json")
+    keep("android.database.Observable")
```

Comment:
Why do we need to keep this?

Reply:
For the unit tests, the fragments we are testing are calling a function in their constructor which at the end of the call stack calls a function in the `android.database.Observable` class which fails with NPE if mocked.